admin_manual/configuration_server/occ_command.rst

```diff
@@ -529,18 +637,51 @@ Files External
 Commands for managing external storage::
 
  files_external
-  files_external:applicable           Manage applicable users and groups for a mount
-  files_external:backends             Show available authentication and storage backends
-  files_external:config               Manage backend configuration for a mount
-  files_external:create               Create a new mount configuration
-  files_external:delete               Delete an external mount
-  files_external:export               Export mount configurations
-  files_external:import               Import mount configurations
-  files_external:list                 List configured mounts
-  files_external:option               Manage mount options for a mount
-  files_external:verify               Verify mount configuration
-
-These commands replicate the functionality in the ownCloud Web GUI, plus two new features:  ``files_external:export`` and ``files_external:import``. Use ``files_external:export`` to export all admin mounts to a text file, and ``files_external:export [user_id]`` to export the mounts of the specified ownCloud user. Use ``files_external:import [filename]`` to import the configuration.
+  files_external:applicable  Manage applicable users and groups for a mount
+  files_external:backends    Show available authentication and storage backends
+  files_external:config      Manage backend configuration for a mount
+  files_external:create      Create a new mount configuration
+  files_external:delete      Delete an external mount
+  files_external:export      Export mount configurations
+  files_external:import      Import mount configurations
+  files_external:list        List configured mounts
+  files_external:option      Manage mount options for a mount
+  files_external:verify      Verify mount configuration
+
+These commands replicate the functionality in the ownCloud Web GUI, plus two new 
+features:  ``files_external:export`` and ``files_external:import``. Use 
+``files_external:export`` to export all admin mounts to a text file, and 
+``files_external:export [user_id]`` to export the mounts of the specified 
+ownCloud user. Use ``files_external:import [filename]`` to import the 
+configuration.
+
+.. _integrity_check_label:
+
+Integrity Check
+---------------
+
+Apps which have an official tag MUST be code signed starting with ownCloud 9.0. Unsigned official apps won't be installable anymore. Code signing is optional for all third-party applications::
+
+ integrity
+  integrity:check-app                 Check an app integrity using a signature.
+  integrity:check-core                Check a core integrity using a signature.
```

Comment:
Remove "a" - "Check core integrity..."
Or is this about a core app's integrity?